documentation/schema.json

```diff
@@ -1125,6 +1118,124 @@
         }
       }
     },
+    "CollectionRuleDefaultsOptions": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "Triggers": {
+          "description": "The Collection Rule Defaults that can be applied to triggers.",
+          "oneOf": [
+            {
+              "type": "null"
+            },
+            {
+              "$ref": "#/definitions/CollectionRuleTriggerDefaultsOptions"
+            }
+          ]
+        },
+        "Actions": {
+          "description": "The Collection Rule Defaults that can be applied to actions.",
+          "oneOf": [
+            {
+              "type": "null"
+            },
+            {
+              "$ref": "#/definitions/CollectionRuleActionDefaultsOptions"
+            }
+          ]
+        },
+        "Limits": {
+          "description": "The Collection Rule Defaults that can be applied to limits.",
+          "oneOf": [
+            {
+              "type": "null"
+            },
+            {
+              "$ref": "#/definitions/CollectionRuleLimitsDefaultsOptions"
+            }
+          ]
+        }
+      }
+    },
+    "CollectionRuleTriggerDefaultsOptions": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "RequestCount": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "description": "The default threshold of the number of requests that start within the sliding window of time; to be used when no other request count is specified.",
```

Comment:
Do we really need to say that "; to be used when no other request count is specified." for all of these options? Isn't that understood when its already said that these are the default values?